src/common/com/intellij/plugins/haxe/model/HaxeClassModel.java

```diff
@@ -356,7 +597,28 @@ public void addField(String name, SpecificTypeReference type) {
     this.getDocument().addTextAfterElement(getBodyPsi(), "\npublic var " + name + ":" + type.toStringWithoutConstant() + ";\n");
   }
 
-  public void addMethod(String name) {
-    this.getDocument().addTextAfterElement(getBodyPsi(), "\npublic function " + name + "() {\n}\n");
+  public void addMethods(HaxeMethodBuilder... methodBuilders) {
+    String out = "";
+    for (HaxeMethodBuilder builder : methodBuilders) {
+      out += "public " + builder.toString() + "\n";
```

Comment:
Probably because initial simplicity. I always try to iterate. That was a fine first approach, that could be improved later.

https://github.com/Akamon/intellij-haxe/commit/78ee5c0714bcb9d98460669488a96656561fdbde